src/gui/sdlmain.cpp

```diff
@@ -1763,10 +1764,15 @@ static void GUI_ShutDown(Section *)
 
 static void SetPriority(PRIORITY_LEVELS level)
 {
-	// TODO replace platform-specific API with SDL_SetThreadPriority
+	// Just let the OS scheduler manage priority
+	if (level == PRIORITY_LEVEL_AUTO)
+		return;
+
+		// TODO replace platform-specific API with SDL_SetThreadPriority
 #if defined(HAVE_SETPRIORITY)
-// Do nothing if priorties are not the same and not root, else the highest
-// priority can not be set as users can only lower priority (not restore it)
+	// If the priorities are different, do nothing unless the user is root,
+	// since priority can always be lowered but requires elevated rights 
+	// to increase
 
 	if((sdl.priority.focus != sdl.priority.nofocus ) &&
 		(getuid()!=0) ) return;
```

Comment:
> The one bit of logic error I can see is if someone does "auto,lowest", it will never change after going to back to focus, so should it be "if one is auto make both auto" and show a message?

That makes sense and will be a big help, as currently the only way for users to realize what's actually being applied (if anything at all) is to inspect the process in their task manager / process list.

Probably should update the conf comment with auto-behavior too; currently it's:
``` text
Pmulti->Set_help("Priority levels for dosbox. Second entry behind the comma is for when dosbox is not focused/minimized.\n"
                 "pause is only valid for the second entry.");
```

Reply:
I think mentioning what `auto` is/does in the above `Set_help(..)` statement is all that's left.

Reply:
```
	Pmulti->Set_help("Priority levels for dosbox. Second entry behind the comma is for when dosbox is not focused/minimized.\n"
	                 "pause is only valid for the second entry. auto disables priority levels and uses OS defaults");
```
?